assets/static/templates.js

```diff
@@ -0,0 +1,67 @@
+var Templates = (function(params) {
+
+
+    var templates = {},
+        config = {
+          // popver with the list of most common labels
+          breakdown: '#breakdown',
+          breakdownContent: '#breakdown-content',
+
+          // reload message if backend version bump is detected
+          reloadNeeded: '#reload-needed',
+
+          // errors
+          fatalError: '#fatal-error',
+          internalError: '#internal-error',
+          updateError: '#update-error',
+
+          // modal popup with label filters
+          modalTitle: '#modal-title',
+          modalBody: '#modal-body',
+
+          // label button
+          buttonLabel: '#label-button-filter',
+
+          // alert group
+          alertGroup: '#alert-group',
+          alertGroupTitle: '#alert-group-title',
+          alertGroupAnnotations: '#alert-group-annotations',
+          alertGroupLabels: '#alert-group-labels',
+          alertGroupElements: '#alert-group-elements',
+          alertGroupSilence: '#alert-group-silence',
+          alertGroupLabelMap: '#alert-group-label-map'
+        }
+
+
+    init = function() {
+        $.each(config, function(name, selector) {
+            try {
+                templates[name] = _.template($(selector).html());
+            } catch (err) {
+                console.error('Failed to parse template ' + name + ' ' + selector);
```

Comment:
Depending on which versions of JavaScript we aim to support it could be written as the following instead:

    console.error(`Failed to parse template ${name} ${selector}`);

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
http://caniuse.com/#feat=template-literals